wpimath/src/test/native/cpp/filter/DebouncerTest.cpp

```diff
@@ -2,47 +2,51 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
-#include "frc/Debouncer.h"  // NOLINT(build/include_order)
+#include <wpi/timestamp.h>
 
-#include "frc/simulation/SimHooks.h"
+#include "frc/filter/Debouncer.h"
 #include "gtest/gtest.h"
+#include "units/time.h"
 
-using namespace frc;
+static units::second_t now = 0_s;
 
 TEST(DebouncerTest, DebounceRising) {
-  Debouncer debouncer{20_ms};
+  WPI_SetNowImpl([] { return units::microsecond_t{now}.to<uint64_t>(); });
+  frc::Debouncer debouncer{20_ms};
 
   debouncer.Calculate(false);
   EXPECT_FALSE(debouncer.Calculate(true));
 
-  frc::sim::StepTiming(100_ms);
+  now += 1_s;
 
   EXPECT_TRUE(debouncer.Calculate(true));
 }
 
 TEST(DebouncerTest, DebounceFalling) {
-  Debouncer debouncer{20_ms, Debouncer::DebounceType::kFalling};
+  WPI_SetNowImpl([] { return units::microsecond_t{now}.to<uint64_t>(); });
+  frc::Debouncer debouncer{20_ms, frc::Debouncer::DebounceType::kFalling};
 
   debouncer.Calculate(true);
   EXPECT_TRUE(debouncer.Calculate(false));
 
-  frc::sim::StepTiming(100_ms);
+  now += 1_s;
 
   EXPECT_FALSE(debouncer.Calculate(false));
 }
 
 TEST(DebouncerTest, DebounceBoth) {
-  Debouncer debouncer{20_ms, Debouncer::DebounceType::kBoth};
+  WPI_SetNowImpl([] { return units::microsecond_t{now}.to<uint64_t>(); });
+  frc::Debouncer debouncer{20_ms, frc::Debouncer::DebounceType::kBoth};
 
   debouncer.Calculate(false);
   EXPECT_FALSE(debouncer.Calculate(true));
 
-  frc::sim::StepTiming(100_ms);
+  now += 1_s;
 
   EXPECT_TRUE(debouncer.Calculate(true));
   EXPECT_TRUE(debouncer.Calculate(false));
 
-  frc::sim::StepTiming(100_ms);
+  now += 1_s;
```

Comment:
Is there a reason the step was changed from 100 ms to 1 s?

Reply:
consistency with other tests; it doesn't matter at all